net-im/signal-desktop-bin/signal-desktop-bin-1.27.2.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/-bin/}"
+
+inherit pax-utils unpacker xdg-utils
+
+DESCRIPTION="Allows you to send and receive messages of Signal Messenger on your computer"
+HOMEPAGE="https://signal.org/
+	https://github.com/WhisperSystems/Signal-Desktop"
+SRC_URI="https://updates.signal.org/desktop/apt/pool/main/s/${MY_PN}/${MY_PN}_${PV}_amd64.deb"
+
+LICENSE="GPL-3 MIT MIT-with-advertising BSD-1 BSD-2 BSD Apache-2.0 ISC openssl ZLIB APSL-2 icu Artistic-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+IUSE=""
+
+RDEPEND="
+	dev-libs/nss
+	media-libs/mesa[X(+)]
+	net-print/cups
+	x11-libs/gtk+:3[X]
+	x11-libs/libXScrnSaver
+	x11-libs/libXtst
+	"
```

Comment:
Weird formatting, truthfully I'm amazed the CI bot ignores this. Could you take the tab out here? Rest looks good.

Reply:
Done